### ISSUE TITLE
Trigger resources not displayed on OpenShift

### ISSUE THREAD

**Issue by a-roberts** [tektoncd/dashboard#919]
Surely introduced by https://github.com/tektoncd/dashboard/issues/815

On our 0.3 release, no check is made to see if the EventListener CRD is known
It is with the 0.4 release

Request details

```
Request URL: https://tekton-dashboard-tekton-pipelines.apps.myopenshiftsystem.ibm.com/proxy/apis/apiextensions.k8s.io/v1beta1/customresourcedefinitions/eventlisteners.tekton.dev
Request Method: GET
Status Code: 404 Not Found
Remote Address: 9.30.254.155:443
Referrer Policy: no-referrer-when-downgrade
```

This could be related but the k8s version is very different (1.13+gke), so assume it's different: https://github.com/tektoncd/dashboard/issues/904

There's nothing for https://tekton-dashboard-tekton-pipelines.apps.myopenshiftsystem.ibm.com/proxy/apis/apiextensions.k8s.io/v1beta1/customresourcedefinitions

Pod error is an `unknown` so I'm gonna guess it's RBAC

**Comment by a-roberts** [comment]
Fixed with

```
- apiGroups:
  - apiextensions.k8s.io
  resources:
  - customresourcedefinitions
  verbs:
  - get
  - list
```
 in the cluster role, PR coming up

**Comment by a-roberts** [comment]
PR https://github.com/tektoncd/dashboard/pull/920

**Comment by a-roberts** [comment]
Merged and in the 0.4.1 release I've just done, closing
